libraries/AP_DroneCAN/AP_DroneCAN.cpp

```diff
@@ -626,7 +647,6 @@ void AP_DroneCAN::SRV_send_himark(void)
 
 void AP_DroneCAN::SRV_send_esc(void)
 {
-    static const int cmd_max = ((1<<13)-1);
```

Comment:
This was only relevant to the ESC output scaling, so it also moved to scale_esc_output